paper/onboarding_issue.md

```diff
@@ -45,11 +45,11 @@ VignetteBuilder: knitr
 https://github.com/bcgov/tidyhydat
 
 - Please indicate which category or categories from our [package fit policies](https://github.com/ropensci/onboarding/blob/master/policies.md#package-fit) this package falls under ***and why**(? (e.g., data retrieval, reproducibility. If you are unsure, we suggest you make a pre-submission inquiry.):
+[e.g., "data extraction, because the package parses a scientific data file format"]
 
   - data retrieval: `tidyhydat` retrieves data from the Water Survey of Canada *datamart* via the `download_realtime_dd()` function from http://dd.weather.gc.ca/hydrometric/csv/
-  - data extraction: The majority of `tidyhydat`'s exported functions provides easy access to Environment Canada's publicly available HYDAT sqlite3 database (http://collaboration.cmc.ec.gc.ca/cmc/hydrometrics/www/) of historical river and lake Canada. HYDAT is updated quarterly and is distributed via the [Canadian Open Government License](https://github.com/bcgov/tidyhydat/blob/master/data-raw/HYDAT_internal_data/LICENSE.OGL-CAN-2.0). HYDAT is not included with this package because it is regularly updated and because the file size is prohibitively large (>1GB) for inclusion in an R package. Rather `tidyhydat` provide the `download_hydat()` function that download it for the user. `download_realtime_ws` provide access via the `httr` package to a new password protected web service. Differences between the download realtime functions are outlined [here](https://github.com/bcgov/tidyhydat/blob/master/vignettes/tidyhydat.Rmd#compare-download_realtime_ws-and-download_realtime_dd)
+  - data extraction: The majority of `tidyhydat`'s exported functions provides easy access to Environment and Climate Change Canada's (ECCC) publicly available HYDAT sqlite3 database (http://collaboration.cmc.ec.gc.ca/cmc/hydrometrics/www/) of historical river and lake Canada. HYDAT is updated quarterly and is distributed via the [Canadian Open Government License](https://github.com/bcgov/tidyhydat/blob/master/data-raw/HYDAT_internal_data/LICENSE.OGL-CAN-2.0). HYDAT is not included with this package because it is regularly updated and because the file size is prohibitively large (>1GB) for inclusion in an R package. Rather `tidyhydat` provides the `download_hydat()` function that downloads it for the user. `download_realtime_ws` provides access via the `httr` package to a new password protected web service provided by ECCC. Differences between the download realtime functions are outlined [here](https://github.com/bcgov/tidyhydat/blob/master/vignettes/tidyhydat.Rmd#compare-download_realtime_ws-and-download_realtime_dd)
```

Comment:
I think the discussion of `download_realtime_ws` belongs in the data retrieval vs data extraction section

Reply:
Actually @ateucher  I'm not quite sure what you mean here. Can you elaborate?

Reply:
Oh nevermind I think I've got it.